libs/financial-aid/shared/src/lib/formatters.ts

```diff
@@ -43,6 +43,25 @@ export const formatPhoneNumber = (phoneNumber: string) => {
 export const formatNationalId = (nationalId: string) =>
   insertAt(nationalId.replace('-', ''), '-', 6) || '-'
 
+export const translateMonth = (month: string) => {
+  const months = [
+    'Janúar',
+    'Febrúar',
+    'Mars',
+    'Apríl',
+    'Maí',
+    'Júní',
+    'Júlí',
+    'Ágúst',
+    'September',
+    'Október',
+    'Nóvember',
+    'Desember',
+  ]
+
+  return months[parseInt(month)]
+}
+
```

Comment:
This could be rewritten like this: 
`export const months = [
     'Janúar',
     'Febrúar',
     'Mars',
     'Apríl',
     'Maí',
     'Júní',
     'Júlí',
     'Ágúst',
     'September',
     'Október',
     'Nóvember',
     'Desember',
   ]`
   
   And then we can find the month with and index.